dev/image/Dockerfile

```diff
@@ -243,14 +234,13 @@ RUN sudo apt-add-repository -y ppa:fish-shell/release-3 && \
     sudo apt install -y fish
 
 # Install tmux + tmuxinator
-RUN brew install tmux tmuxinator
-
-# Install redis-server
-RUN brew install redis@7.0
-
-# Install zed & spicedb CLI
-RUN brew install authzed/tap/zed
-RUN brew install authzed/tap/spicedb
+RUN brew install tmux tmuxinator \
+    # Install redis-server
+    && brew install redis \
```

Comment:
Redis is no longer versioned with this change

Reply:
Yes, the version right now is 7.2.1. I tried to use @7, @7.2 but that is not supported